app/src/main/java/campus/tech/kakao/map/LocationAdapter.kt

```diff
@@ -0,0 +1,44 @@
+package campus.tech.kakao.map
+
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.TextView
+import androidx.recyclerview.widget.RecyclerView
+
+class LocationAdapter(
+    var locationList: ArrayList<LocationData>,
```

Comment:
음 constructor parameter 로 받을 것이라면 List<LocationData> 로 immutable 하게 받으심이 좋을 것 같아요!
```suggestion
    val locationList: List<LocationData>,
```